backend/src/akvo/lumen/lib/resource_impl.clj

```diff
@@ -0,0 +1,41 @@
+(ns akvo.lumen.lib.resource-impl
+  (:require [akvo.lumen.util :refer [squuid]]
+            [hugsql.core :as hugsql]
+            [ring.util.response :refer [not-found response]])
+  (:import [java.sql SQLException]))
+
+(hugsql/def-db-fns "akvo/lumen/lib/resource.sql")
+
+
+(defn current-plan [tenant-conn]
+  (if-let [r (select-plan tenant-conn)]
+    r
+    {"tier" "planless"}))
```

Comment:
I wonder if we should have a default tier for existing tenants instead of "planless"?